backends-velox/src/test/scala/io/glutenproject/execution/FallbackSuite.scala

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.glutenproject.execution
+
+import io.glutenproject.GlutenConfig
+
+import org.apache.spark.SparkConf
+import org.apache.spark.sql.execution.{SparkPlan, VeloxColumnarToRowExec}
+import org.apache.spark.sql.execution.adaptive.AdaptiveSparkPlanHelper
+
+class FallbackSuite extends WholeStageTransformerSuite with AdaptiveSparkPlanHelper {
```

Comment:
why this suite only for  velox backend ?

Reply:
We have ut in `FallbackStrategiesSuite`. This test suite is due to I face a velox backend bug and I think I should leave some end-to-end tests here. I do not have any background/env about CH, so if you think CH backend should have these, then you can add some in followup.

Reply:
https://github.com/oap-project/gluten/issues/2829